sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -262,7 +262,7 @@ Language=Polish
 Usługa %1 nie mogła skonfigurować źródła zdarzenia.
 .
 Language=Romanian
-Serviciul %1 nu a putut configura o sursă a evenimentului.
+Serviciul %1 nu a putut configura o sursă a unui eveniment.
```

Comment:
To be frank, "sursă a unui eveniment" sounds worse than "sursă de eveniment" which [you've chosen before](https://github.com/reactos/reactos/pull/5431#discussion_r1262780104). The improvement I was hinting at was "sursă de evenimente".